Felienne/hedy
# [BUG FIX] Only redirects users to landing page at first login

**Description**
Due to a bug users are currently always re-directed to the landing page after login (when not a teacher). This is undesirable and in this PR we fix this by added a `first time` value to the `afterLogin()` function. Keeping tracking of the user login and making sure that only the first time is re-directed to the landing-page.

**Fixes**
No related issue.

**How to test**
Create a new account and verify that you are automatically re-directed to the landing page. Log out and in again and verify that in this case you are re-directed to the programs page.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
